lib/block-supports/layout.php

```diff
@@ -170,6 +187,7 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 			return $block_content;
 		}
 		$used_layout = $default_layout;
+		$padding     = isset( $default_layout['padding'] ) ? $default_layout['padding'] : null;
```

Comment:
This adds padding to all default layout blocks, which is not the current behaviour in trunk, so it might break some stuff.